.nvmrc

```diff
@@ -1 +1 @@
-4.4.0
+lts/argon
```

Comment:
Crap, you're right. I'm splitting the stuff out into 2 separate PRs so I'll drop this change.